### DISCUSSION TITLE
About mapping character to a specific glyphs

### DISCUSSION THREAD

**Discussion by Vincent-Maximon** [SilenceLeo/snowb-bmf#80] [Q&A]
- First of all, thanks for your great work!

I am hereby to seek for you help in mapping a characters into an imported glyph as i remember so far that an older version allow me to manually map my character into a glyph that i imported but now i cannot do it anymore. Can you teach me how to do that as it can improve the speed of working with bitmap font for me :)))

**Comment by SilenceLeo** [comment]
I fixed this. https://github.com/SilenceLeo/snowb-bmf/issues/82

Please try again.
